app/models/event.rb

```diff
@@ -1,34 +1,65 @@
 # frozen_string_literal: true
 
+# Model of events created by the user
 class Event < ApplicationRecord
   mount_uploader :event_avatar, AvatarUploader
-  mount_uploader :event_box_office, SpreadsheetUploader
   belongs_to :user
-  # <!--===================-->
-  # <!--to add nested scaffold-->
+
   has_many :seats, dependent: :destroy
   has_many :guests, dependent: :destroy
   has_many :email_services, dependent: :destroy
   has_many :referrals, dependent: :destroy
-  # <!--===================-->
-  def calculate_seating_summary(event_id)
+  def calculate_seating_summary(ticket_sales)
```

Comment:
Sorry for the late review. Please add rspec for this method.
<img width="1413" alt="Screenshot 2024-11-06 at 6 23 37 PM" src="https://github.com/user-attachments/assets/be99cfbf-2a5d-4e76-9db4-f9bffc7c98d4">
You can open the file `coverage/index.html` locally to see the report after running rspec.